boards/DIM/Src/Io/Io_Adc.c

```diff
@@ -0,0 +1,28 @@
+#include <stm32f3xx.h>
+#include "Io_SharedAdc.h"
+#include "Io_Adc.h"
+
```

Comment:
Should we include a small comment about listing out the elements in the enum in ascending order, exactly like how its specified in Cube?

Reply:
What do you mean asecnding order?

Reply:
Seems like this was addressed in your latest comment